scripts/cloud-api-demo/reset_mongo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+MONGO_CONTAINER_NAME="talawa-api-mongodb-1"
```

Comment:
Add comments regarding the purpose of this script file

Reply:
Added